pkgs/by-name/cr/crabfit-frontend/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  yarn,
+  fixup_yarn_lock,
+  google-fonts,
+  api_url ? "http://127.0.0.1:3000",
+  frontend_url ? "crab.fit",
```

Comment:
I see why your adding the url as a option parameter, but it's somewhat ugly because everyone will have to rebuild the package. And it would be preferable to patch some way of passing this as some kind of runtime option like a env var or in a configuration file.

Reply:
It would require overhauling the i18n system, I think this is out of scope for a first version, but we should have an issue documenting what we need here and make use of runtime variables for the i18n system.

Reply:
I'm fine with doing this in a later version, but I think we should open a issue upstream about it and link it here.

Reply:
@Tom-Hubrecht would you mind opening a issue upstream?